base/ltoutput.dtx

```diff
@@ -2350,9 +2402,18 @@
 %    Then the footnotes are appended:
 %    \begin{macrocode}
      \@outputbox@append{%
+%    \end{macrocode}
+%    First we call a socket which may or may not back up by the depth
```

Comment:
"may or may not" is confusing here! 

Maybe:
. . . socket which controls whether or not we backup by . . .